lib/bitcoin/versions/latest/NormalizedFeeCalculator.ts

```diff
@@ -21,17 +22,26 @@ export default class NormalizedFeeCalculator implements IFeeCalculator {
   }
 
   public async getNormalizedFee (block: number): Promise<number> {
+    // DB call optimization
+    // https://github.com/decentralized-identity/sidetree/issues/936
```

Comment:
Nit: standard patterns is to append `TODO` to enable searching and automated tooling (such as extension in VSC).
```suggestion
    // TODO: https://github.com/decentralized-identity/sidetree/issues/936
```